flight/Modules/Autotune/autotune.c

```diff
@@ -191,6 +193,8 @@ static void AutotuneTask(void *parameters)
 		const uint32_t PREPARE_TIME = 2000;
 		const uint32_t MEASURE_TIME = 60000;
 
+		static uint32_t updateCounter = 0;
```

Comment:
I think 2 in this case, especially since the file already has quite a bit of _ action. Agree that we should come up with a general rule.